sdk/core/azure-core/inc/azure/core/http/policies/policy.hpp

```diff
@@ -363,11 +372,14 @@ namespace Azure { namespace Core { namespace Http { namespace Policies {
     /**
      * @brief HTTP retry policy.
      */
-    class RetryPolicy
-#if !defined(TESTING_BUILD)
-        final
+    class RetryPolicy _azure_NON_FINAL_FOR_TESTS : public HttpPolicy {
+
+#if defined(_azure_TESTING_BUILD)
+      // make tests classes friends to validate RetryPolicy
```

Comment:
From the naming it is pretty clear, why inside "ifdef testing" we have "friend RetryPolicyTest", so I'd remove the comment.

Reply:
Agreed, was following conventions from elsewhere.